lib/Runtime/Library/TypedArray.cpp

```diff
@@ -1730,6 +1730,24 @@ namespace Js
         return JavascriptArray::CopyWithinHelper(nullptr, typedArrayBase, typedArrayBase, length, args, scriptContext);
     }
 
+    Var TypedArrayBase::GetKeysEntriesValuesHelper(Arguments& args, ScriptContext *scriptContext, LPCWSTR apiName, JavascriptArrayIteratorKind kind)
```

Comment:
I am inclined to leave this as is. It does few other operations than generic check. I am working on second iteration (which has some other fixes) which has some other changes, I can make few adjustment to that function itself. Is that okay?

---
In reply to: [79927530](https://github.com/Microsoft/ChakraCore/pull/1618#discussion_r79927530) [](ancestors = 79927530)